cime_config/tests.py

```diff
@@ -95,6 +95,15 @@
             )
         },
 
+    "eam_condidiag" : {
+        "tests"   : (
+            "SMS_D_Ln5.ne4_oQU240.F2010",
+            "SMS_D_Ln5.ne4_oQU240.F2010.eam-condidiag_dcape",
```

Comment:
@huiwanpnnl , can you please add one of the tests (e.g., e3sm_atm_extra_coverage) to the e3sm_atm_extra_coverage test suite in order to be tested regularly?

Reply:
@wlin7, I added the `eam_condidiag` suite to `e3sm_atm_extra_coverage` in [77851b0](https://github.com/E3SM-Project/E3SM/pull/5509/commits/77851b0dc0b4e0bcb92626f5f7f92765755e8267). Is this what you meant?

Reply:
@huiwanpnnl , I was thinking just one of the tests for general condidiag logic. But if you think meaningful to protect the condidiag  codes for both dcape and rhi, sounds good to add to extra_coverage for both.

Reply:
Great. Then let's leave it this way. I would like to have the dCAPE calculations tested as they use a subroutine in `zm_conv.F90` that the convection experts might update; the RHI example tests conditional sampling, so it adds value, too.